packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -378,24 +378,38 @@ describe('plugin-meetings', () => {
     });
 
     describe('joinMeeting', () => {
-      it('#Should call `meetingRequest.joinMeeting', async () => {
-        const meeting = {
+      const joinMeetingResponse = {
+        body: {
+          mediaConnections: [],
+          locus: {
+            url: 'differentLocusUrl',
+            self: {
+              id: 'selfId',
+            },
+          },
+        },
+        headers: {
+          trackingid: 'trackingId',
+        },
+      }
+      let meeting
```

Comment:
I am not sure if syntaxys here is right. Should ";" be at the end like in other test cases?
I feel that some js files in SDK have an issue with checking this, but still, I believe we need to try to align this